web/impact/impact/v1/views/office_hours_calendar_view.py

```diff
@@ -283,18 +300,9 @@ def succeed(self):
 
 
 def _location_lookups(location_path):
-    location_fields = (
-        'id',
-        'street_address',
-        'timezone',
-        'country',
-        'state',
-        'name',
-        'city',
-    )
     return dict([("location_" + field, F("{}__{}".format(
         location_path, field)))
-                 for field in location_fields])
+        for field in LOCATION_FIELDS])
```

Comment:
Why are we outdenting this?

Reply:
@geofrocker It might be worth extracting the guts of this comprehension as a function to make the formatting work better:

```
def _location_lookups(location_path):
    return dict([location_dict_pair(field, location_path)
                 for field in LOCATION_FIELDS])

def _location_dict_pair(field, location_path):
    return (_format_key(field), _make_f_exprettion(field, location_path))

def _format_key(field):
    return "location_" + field

def _make_f_expression(field, location_path):
    return F("{}__{}".format(location_path, field))
```